src/smart-components/workspaces/workspace-detail/AssetsCards.tsx

```diff
@@ -1,9 +1,82 @@
+import {
+  Brand,
+  Card,
+  CardBody,
+  CardFooter,
+  CardHeader,
+  CardTitle,
+  Gallery,
+  Icon,
+  Panel,
+  PanelHeader,
+  PanelMain,
+  PanelMainBody,
+} from '@patternfly/react-core';
+import { ArrowRightIcon } from '@patternfly/react-icons';
+import Messages from '../../../Messages';
 import React from 'react';
+import { useIntl } from 'react-intl';
 
-interface AssetsCardsProps {}
+interface AssetsCardsProps {
+  workspaceName: string;
+}
 
-const AssetsCards: React.FunctionComponent<AssetsCardsProps> = () => {
-  return <>ASSETS CARDS</>;
+const AssetsCards: React.FunctionComponent<AssetsCardsProps> = ({ workspaceName }: AssetsCardsProps) => {
+  const InsightsIcon = '/apps/frontend-assets/console-landing/insights.svg';
+  const OpenShiftIcon = '/apps/frontend-assets/console-landing/openshift.svg';
+  const InsightsNavURL = `/insights/inventory?workspace=${workspaceName}`;
+  const OpenShiftNavURL = `/openshift/?workspace=${workspaceName}`;
+  const intl = useIntl();
+
+  return (
+    <Panel>
+      <PanelHeader>{intl.formatMessage(Messages.assetManagementOverview)}</PanelHeader>
+      <PanelMain>
+        <PanelMainBody>
+          <Gallery
+            hasGutter
+            minWidths={{
+              md: '100px',
+              lg: '150px',
+              xl: '200px',
+              '2xl': '300px',
+            }}
+          >
+            <Card>
+              <CardHeader>
+                <Brand src={InsightsIcon} alt="Insights logo" style={{ width: '48px' }} />
+              </CardHeader>
+              <CardTitle>{intl.formatMessage(Messages.assetManagementInsights)}</CardTitle>
+              <CardBody>{intl.formatMessage(Messages.assetManagementInsightsOverview)}</CardBody>
+              <CardFooter>
+                <a href={InsightsNavURL}>
+                  {intl.formatMessage(Messages.assetManagementInsightsNav)}
+                  <Icon className="pf-v5-u-ml-sm" isInline>
+                    <ArrowRightIcon />
+                  </Icon>
+                </a>
```

Comment:
We should probably wrap these links with Button with component set to `a` and onClick to use navigate function. This way the page would not force refreshed and only navigated to the proper screen.